app/code/Magento/Backend/etc/adminhtml/system.xml

```diff
@@ -203,6 +203,21 @@
                     <backend_model>Magento\Config\Model\Config\Backend\Image\Adapter</backend_model>
                     <comment>When the adapter was changed, please flush Catalog Images Cache.</comment>
                 </field>
+                <field id="max_quality" translate="label comment" type="text" sortOrder="100" showInDefault="1" showInWebsite="0" showInStore="0" canRestore="1">
```

Comment:
Here you're just adding a new configuration field, I don't see the implementation/usage of this field.

Reply:
Actually i try to use max quality for images, but i couldn't find how to use that in magento 2